packages/app/src/styles/theme/_apply-colors.scss

```diff
@@ -20,6 +20,7 @@ $bgcolor-page-list-group-item-active: lighten($primary, 76%) !default;
 $color-page-list-group-item-meta: $gray-500 !default;
 $color-search-page-list-title: $color-global !default;
 $bgcolor-subnav: darken($bgcolor-global, 3%) !default;
+$bgcolor-list-hover: darken($primary, 8%) !default;
```

Comment:
`$bgcolor-list-hover` should not be declared on this file since the bgColor need to change by light/dark modes.

the background colors should be like  Pagetree.